File: Python3/Secuenciales/ej6/ej6.py
# -*- coding: utf-8 -*-
# Calcular la media de tres números pedidos por teclado.
#1.pedir numero1
#2.almacenar numero1
#3.pedir numero2
#4.almacenar numero2
#5.pedir numero3
#6.almacenar numero3

print("== Calcular media de tres números ==")
numero1 = float(input("introducir primer número1> "))
numero2 = float(input("introducir primer número2> "))
numero3 = float(input("introducir primer número3> "))
media = ((numero1+numero2+numero3)/3)
print("La media aritmética de "+str(numero1)+", "+str(numero2)+" y "+str(numero3)+" es "+ str(media))

File: Java/Secuenciales/ej6.java
import java.util.Scanner;

/**
*  Calcular la media de tres números pedidos por teclado
*  @author Alberto Miguel Martinez Jimenez
*/

//Almacenar numero1
//Almacenar numero2
//Almacenar numero3
//sumar los numeros
//dividir entre tres
//mostrar la media

public class ej6 {
	public static void main(String[] args) {
		double sumatorio;
		double media;
		Scanner scanner = new Scanner(System.in);//Intanciar el objeto Scanner
		
		System.out.println("Introduce primer número> ");
		sumatorio = scanner.nextDouble();
		
		System.out.println("Introduce segundo número> ");
		sumatorio += scanner.nextDouble();
		
		System.out.println("Introduce tercer número> ");
		sumatorio += scanner.nextDouble();
		
		media = sumatorio/3;
		
		System.out.println("La media es "+media);
	}
}